datus-clickzetta/datus_clickzetta/connector.py

```
@@ -528,12 +547,14 @@ def execute_queries_arrow(self, queries: List[str]) -> List[ExecuteSQLResult]:
                     command_df = self._run_command(query)
                     row_count = self._extract_row_count(command_df)
                     results.append(
-                        ExecuteSQLResult(success=True, data=None, row_count=row_count)  # No data for non-SELECT queries
+                        ExecuteSQLResult(success=True, sql_return=None, sql_query=query, row_count=row_count)
                     )
             except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)